wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wdl/WdlWriterImpl.scala

```diff
@@ -465,6 +465,7 @@ object WdlWriterImpl {
           case _: Zip => functionCall("zip")
           case _: Cross => functionCall("cross")
           case _: Prefix => functionCall("prefix")
+          case _: Suffix => functionCall("suffix")
```

Comment:
This would be a great time to consider dropping the WDL Writer, which was created to help users upgrade from draft-2 to 1.0... and was my first major project on Cromwell back in 2018!

Reply:
Whoa! Let's discuss tomorrow.